addons/resource.language.en_gb/resources/strings.po

```diff
@@ -20394,3 +20394,14 @@ msgstr ""
 msgctxt "#39010"
 msgid "Select sort method"
 msgstr ""
+
+#: system/settings/settings.xml
+msgctxt "#39011"
+msgid "Wait for network after waking up"
+msgstr ""
+
+#. Description of setting with label #39011 "Wait for network after waking up"
+#: system/settings/settings.xml
+msgctxt "#39012"
+msgid "Set the maximum time to wait for the network to come up after waking up."
```

Comment:
"Set the maximum time to wait for the network to come up after starting or waking up. When time has past before network is up startup will continue."
If it indeed also handles a startup (which it should imo)